src/System.Resources.Writer/src/System/Resources/ResourceWriter.cs

```diff
@@ -116,26 +129,11 @@ public void AddResource(string name, object value)
             }
         }
 
-        // Adds a resource of type Stream to the list of resources to be 
-        // written to a file.  They aren't written until Generate() is called.
-        // Doesn't close the Stream when done.
-        //
-        public void AddResource(string name, Stream value)
```

Comment:
>AddResource [](start = 20, length = 11)

would be any compat concern removing this method and making the other with the optional parameter?

Reply:
This isn't actually removed.  We're just moving it to the partial file.  I don't have it on my new type.

Reply:
And yes, we cannot remove it and be binary compatible.  It'd be source compatible but missing the method is a binary breaking change.

Reply:
Thanks. This is clear now.